### ISSUE TITLE
[Bug] curl with SSL on a 2019 mac that hasn't been fully updated

### ISSUE THREAD

**Issue by benatkin** [webinstall/webi-installers#301]
## What were you trying to install (or what else went wrong)?

webi itself

## What exactly did you do?

I ran `curl -sS https://webinstall.dev/webi | bash`

## What went wrong?

I got this error:

```
❯ curl -sS https://webinstall.dev/webi | bash
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

## Which OS did you try on?

- [x] Mac
- [ ] Windows
- [ ] Linux

## What type of computer (i.e. laptop, desktop, Raspberry Pi)?

- [x] Desktop

---

I'm also getting this with [dprint.dev](https://dprint.dev/install/) - it loads in the browser fine, and curl is working for other things.

Both are on Let's Encrypt. [I think this might be related.](https://community.letsencrypt.org/t/problem-with-certificate-has-expired/161013/4)

I've missed some updates on my mac because I only have 128GB and haven't freed up enough space to run the updates. Running Catalina 10.15.5.

**Comment by coolaj86** [comment]
Sounds like you need a static version of curl (though you may run into other problems later).

- Linux https://github.com/moparisthebest/static-curl
- Windows https://curl.se/windows/
- macOS ... no idea

If there were a business use case for this I'd say I could run a mirror site with certificates bought from companies with older certificate chains, but... I don't think this is that sort of scenario.

My one suggestion is to try to get `wget` and use that instead. Maybe this helps? https://trick77.com/wget-binary-for-mac-os-x/

**Comment by benatkin** [comment]
Thanks. I think maybe LetsEncrypt has prescribed a different web server config than your site and dprint are using. But yeah the root cause is probably an old root store. I'll keep looking at it.

**Comment by benatkin** [comment]
It just works with curl for me now!

I ran just the XCode Developer Tools update, had to click a small link below the upgrade for Big Sur. The update said it failed, but I think it partially updated, maybe completely. I ran it twice and think it took longer before it failed the second time, so maybe the second time is when the update got curl to work with webi and dprint. I also upgraded my homebrew packages, so maybe that was it.

A closed issue that can be found by searchers is probably good because this is a tiny % of users. If it isn't super tiny, an FAQ entry.
